app/views/users/dashboard.html.erb

```diff
@@ -3,39 +3,63 @@
 <div class="container justify-content-around">
     <div class="row"> 
         <div class="col-9">
-            <table class="table table-hover">
-                <thead>
-                    <tr>
-                        <th scope="col">#</th>
-                        <th scope="col">#</th>
-                        <th scope="col">#</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <th scope="row">1</th>
-                        <td>#</td>
-                        <td>#</td>
-                    </tr>
-                    <tr>
-                        <th scope="row">2</th>
-                        <td>#</td>
-                        <td>#</td>
-                    </tr>
-                </tbody>
-            </table>
+            <h3>activities</h3>
+                <% @activities.each do |activity| %>
+                  <div class="media border-top p-2">
+                    <div class="media-body row">
+                      <div class="img col-md-2">
+                        <% if User.find(activity.user_id).image? %>
```

Comment:
@YuheiTsunoda Use a helper to organize your image resource:

In your **users_helper.rb** file
```
def user_image(user)
    path = user.image? ? user.image.url : "default.jpg"

    image_tag(path)
  end
```
To use this helper:
`user_helper(activity.user)`